pkgs/molior-client-http/usr/lib/molior/build-script

```diff
@@ -21,6 +21,14 @@ if [ -z "$MOLIOR_SERVER" ]; then
   MOLIOR_SERVER=172.16.0.254
 fi
 
+# Workaround obsolete pxz package on buster
+xzversion=`dpkg -s xz-utils | grep ^Version: | sed 's/^Version: //'`
+if dpkg --compare-versions "$xzversion" lt 5.2.4-1; then
+  TAR_PXZ="-Ipxz"
+else
+  TAR_PXZ=""
```

Comment:
why not using "xz -T 0" for compression/decompression?

Reply:
in order to not use 100% CPU ;)